### PR TITLE
[In Development] added new Settings window for repositories configuration

### PR DESCRIPTION
**Pull request by vadkostuk**

### CODE REVIEW

File: GitTank/Dto/ConfigurationDto.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections.Generic;
+
+namespace GitTank.Dto
+{
+    public class ConfigurationDto
+    {
+        public AppSettings appSettings { get; set; }
+    }
+
+    public class AppSettings
```
Comment (TsapAndriy) [inline GitTank/Dto/ConfigurationDto.cs:10]: Please provide separate files for each classes

File: GitTank/Dto/ConfigurationDto.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    {
+        public List<SourcePathes> sourcePathes { get; set; }
```
Comment (TsapAndriy) [inline GitTank/Dto/ConfigurationDto.cs:12]: Public property names should start from Upper case. Here and below.

File: GitTank/GitProcessor.cs [hunk 3]
```diff
@@ -101,6 +101,13 @@ public async Task<string> Branches()
             return await _processHelper.Execute();
         }
 
+        public string GetAllBranches(string repositoryPath)
```
Comment (kseniadovhanyk) [inline GitTank/GitProcessor.cs:104]: why not async?

File: GitTank/ViewModels/SettingsViewModel.cs [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+using System.Linq;
+using System.Windows.Forms;
+
+namespace GitTank.ViewModels
+{
+    public class SettingsViewModel : BaseViewModel
+    {
+        private readonly IConfiguration _configuration;
+        private readonly GitProcessor _gitProcessor;
+        private bool _isAddRepositoryButtonEnable = true;
```
Comment (kseniadovhanyk) [inline GitTank/ViewModels/SettingsViewModel.cs:17]: typo, missing letter d in the end

**Review by Yuriy-Pelekh — APPROVED** [review]

**Pull request merged**